test/Microsoft.AspNetCore.Cors.Test/CorsServiceTests.cs

```diff
@@ -881,6 +881,24 @@ public void ApplyResult_PreflightMaxAge_MaxAgeHeaderAdded()
             Assert.Equal("30", httpContext.Response.Headers["Access-Control-Max-Age"]);
         }
 
+        [Fact]
+        public void EvaluatePolicy_MultiOriginsPolicy_ReturnsVaryByOriginHeader()
```

Comment:
I believe One origin is handled in other test cases, like EvaluatePolicy_NoMatchingOrigin_ReturnsInvalidResult
This test case handles Multiple Origins.

And I've added the negative test case EvaluatePolicy_MultiOriginsPolicy_NoMatchingOrigin_ReturnsInvalidResult